eiffel-vocabulary/EiffelServicePausedEvent.md

```diff
@@ -0,0 +1,20 @@
+
+# EiffelServicePausedEvent (ArtS)
+
+The EiffelServicePausedEvent declares that a service has been paused .
```

Comment:
In a change management context we pause the service to push a change to the configuration.
__Description:__ The EiffelServicePausedEvent declares that a service has been set in Paused State intentionally by a human or a machine. The preceeding event state is the EiffelServiceStartedEvent.The subsequent event state is the EiffelServiceResumedEvent.